python/paddle/fluid/contrib/tests/test_calibration.py

```diff
@@ -120,13 +120,13 @@ class TestCalibration(unittest.TestCase):
     def setUp(self):
         # TODO(guomingz): Put the download process in the cmake.
         # Download and unzip test data set
-        imagenet_dl_url = 'http://paddle-inference-dist.bj.bcebos.com/int8/calibration_test_data.tar.gz'
+        imagenet_dl_url = 'http://paddle-inference-dist.cdn.bcebos.com/int8/calibration_test_data.tar.gz'
```

Comment:
@hshen14 Please cache the downloading and unzipping in next PR. Since CI random fails on downloading.
![image](https://user-images.githubusercontent.com/6836917/51786284-ec06e380-219c-11e9-9936-d59751b38b16.png)
http://ci.paddlepaddle.org/viewLog.html?tab=buildLog&buildTypeId=Paddle_PrCiNight&buildId=53510

You can use the `download` function in `python.paddle.dataset.common`, like
https://github.com/PaddlePaddle/Paddle/blob/88bd7e1a6194a123237129ee4e9cc5c70516946c/python/paddle/dataset/flowers.py#L226
```
download(URL, 'int8/xxx', DATA_MD5)
```
It will be cached in `~/.cache/paddle/dataset/int8/xxx`
https://github.com/PaddlePaddle/Paddle/blob/88bd7e1a6194a123237129ee4e9cc5c70516946c/python/paddle/dataset/common.py#L38